packages/core-mobile/app/store/account/listeners.ts

```diff
@@ -156,7 +160,7 @@ export const addAccountListeners = (
   })
 
   startListening({
-    matcher: isAnyOf(toggleDeveloperMode),
+    matcher: isAnyOf(toggleDeveloperMode, setActive),
```

Comment:
I don't think we need to reloadAccounts whenever a new network is set to active. we just need to do it when it is devnet right? if so, let's use a matcher here to make sure reload is only called when it is devnet.

Reply:
we need to reload account when we switch to devnet or from devnet to another network.